### PR TITLE
Correcting names of modules and new service creation

### PR DESCRIPTION
**Pull request by o-marshmallow**
Updating tutorials to the current dev version

### CODE REVIEW

File: src/tutowidgets.wiki [hunk 1]
```diff
@@ -197,7 +201,11 @@ let mywidget s1 s2 =
   div ~a:[a_class ["mywidget"]] [button; content]
 
 let _ =
-  Ex_app.register_service ~path:[] ~get_params:Eliom_parameter.unit
+  Ex_app.register
+    (Eliom_service.create
+       ~id:(Eliom_service.Path [])
+       ~meth:(Eliom_service.Get Eliom_parameter.unit)
+       ())
```
Comment (vasilisp) [inline src/tutowidgets.wiki]: Why not `Ex_app.create`, which creates a service and registers it with the same function? (Same issue repeated later.)

**Comment by vasilisp** [comment]
Sorry, we somehow overlooked this PR :).

**Comment by o-marshmallow** [comment]
No problem. I didn't know there was such a function, that is the first way I found to make it work when I was learning how to use ocsigen with the given examples :)

**Comment by vasilisp** [comment]
Can you rebase the PR on top of `master` if still relevant? Also, if you can update for `create` as discussed above, that would be great.

**Comment by o-marshmallow** [comment]
This PR is not relevant anymore, it has already been corrected in the branch master

**Pull request closed without merging**